WordPress/WordPressTest/EUUSCompliance/CompliancePopoverViewModelTests.swift

```diff
@@ -0,0 +1,87 @@
+import XCTest
+@testable import WordPress
+
+final class CompliancePopoverViewModelTests: CoreDataTestCase {
```

Comment:
Thank you for adding tests for the view model!
What about adding some tests for the coordinator, the issue reported [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/21012#discussion_r1253802271) would've been caught by tests.